tests/drivers/uart/uart_async_api/boards/atsamr21_xpro.overlay

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&sercom3 {
+	status = "okay";
+	compatible = "atmel,sam0-uart";
+	current-speed = <115200>;
+
+	/* Internally loop-back TX and RX on PAD0 */
+	rxpo = <0>;
+	txpo = <0>;
+
+	/* Configure DMA channels for async operation */
+	rxdma = <0>;
+	txdma = <1>;
```

Comment:
This configuration works with code on Zephyr master. After #24572 be merged this need be updated to:
dma-names = "tx", "rx";
dmas = <&dmac 1 0x8>, <&dmac 0 0x7>;